DDL/transaction DDL.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE transaction                                                          
+(                                                                              
+        --Primary Key                                                                                         
+        PRIMARY KEY (transaction_id),                                             
+                                                                                 
+        --Columns                                                              
+        transaction_id INT(10) NOT NULL GENERATED ALWAYS AS IDENTITY UNIQUE,
+		description VARCHAR(50) NULL,
+		time_stamp TIMESTAMP,
```

Comment:
Needs to be NOT NULL and should default to current timestamp. Maybe also name change.